File: d66-social/src/main/java/com/eroldme/d66/subreddit/subreddit/SubredditRepository.java
package com.eroldme.d66.subreddit.subreddit;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

/**
 * @author Mompati 'Patco' Keetile
 * @created 19-12-2022 @ 12:34
 */
@Repository
public interface SubredditRepository extends JpaRepository<Subreddit, Long> {
}

File: d66-social/src/main/kotlin/com/eroldme/d66/subredit/subredit/SubredditRepository.kt
package com.eroldme.d66.subredit.subredit

import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

/**
 * @author Mompati 'Patco' Keetile
 * @created 06-12-2022 @ 14:13
 */
@Repository
interface SubredditRepository : JpaRepository<Subreddit, Long>
